chapters/managementCompanies.md

```diff
@@ -0,0 +1,19 @@
+# Companies management
+## Show all companies
+The *Show Company* button in the menu bar allows you to give access to all companies recorded in MaaS. The list of companies will be shown in tabular form.
```

Comment:
The *Show Companies* button (nell'immagine è al plurale).
@drigoni: devi spiegarmi cosa intendi con questa frase perché non è molto chiara.